scalableminds/webknossos
# Avoid duplicate layers when applying suggestions

### Steps to test:
- on disk, duplicate a layer of a dataset
- open dataset settings from dashboard, click accept selection, save
- new layer should be included, no duplicate assertion should show

### Issues:
- fixes #5401 

------
